### DISCUSSION TITLE
Hola soy nuevo en igithub

### DISCUSSION THREAD

**Discussion by 4lexxe** [community/community#68281] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hola mundo, soy nuevo en github espero poder divertirme y aprender junto a ustedes en esta comunidad. 😊👌

**Comment by HyNuwa** [comment]
Hola mundo cruel

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
